### PR TITLE
Added info HTTP endpoint.

### PR DESCRIPTION
**Pull request by nikhilvibhav**
Ref. issue #20
- added info HTTP endpoint which returns a JSON body with info on the service.
- now loads the rest api url, port, etc. from the properties file.

### CODE REVIEW

File: src/main/java/com/capgemini/brahma/examples/route/InfoRoute.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    private String restApiBaseUrl;
+
+    @Value("${rest.api.port}")
+    private String restApiPort;
+
+    @Override
+    public void configure() throws Exception {
+        Map<String, String> infoMap = new HashMap<>();
+
+        String infoName = "Brahma Camel-Boot";
```
Comment (andrewharmellaw) [inline src/main/java/com/capgemini/brahma/examples/route/InfoRoute.java]: Can we make this a config property too?

File: src/main/java/com/capgemini/brahma/examples/route/InfoRoute.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        String infoDescription = "A seed-project for starting new REST-API Camel/Spring Boot projects";
```
Comment (andrewharmellaw) [inline src/main/java/com/capgemini/brahma/examples/route/InfoRoute.java]: And this?  Loaded from file like the others.  Then when you create a new project from a brahma-seed you edit all the specifics for your new service in one place (the config file)

**Comment by nikhilvibhav** [comment]
`infoName` and `infoDescription` are now being loaded from the config file.

File: config/application.properties [hunk 3]
```diff
@@ -1,4 +1,9 @@
 spring.main.sources = com.capgemini.brahma.Application
+
+# For Info Route
```
Comment (andrewharmellaw) [inline config/application.properties:3]: Nice.

File: src/main/java/com/capgemini/brahma/examples/route/ExampleRestDslRoute.java [hunk 4]
```diff
@@ -9,31 +9,31 @@
...
+    @Value("${rest.api.description}")
     private String restApiDesc;
+
+    @Value("${rest.api.port}")
+    private String restApiPort;
     
     @Override
     public void configure() throws Exception {
 
-        // TODO: this should be loaded from config
```
Comment (andrewharmellaw) [inline src/main/java/com/capgemini/brahma/examples/route/ExampleRestDslRoute.java:27]: Great. Thanks.

**Pull request merged**